hack/e2e.sh

```diff
@@ -2,9 +2,26 @@
 
 set -euo pipefail
 
+OPENSHIFT_CI=${OPENSHIFT_CI:-""}
+E2E_RELEASE_BRANCH=${E2E_RELEASE_BRANCH:-""}
```

Comment:
Nit, I think the following has the same result
```suggestion
OPENSHIFT_CI=${OPENSHIFT_CI:-}
E2E_RELEASE_BRANCH=${E2E_RELEASE_BRANCH:-}
```